onebusaway-android/src/main/java/org/onebusaway/android/map/RouteMapController.java

```diff
@@ -211,6 +212,33 @@ public void onSaveInstanceState(Bundle outState) {
         outState.putString(MapParams.ROUTE_ID, mRouteId);
         outState.putBoolean(MapParams.ZOOM_TO_ROUTE, mZoomToRoute);
         outState.putBoolean(MapParams.ZOOM_INCLUDE_CLOSEST_VEHICLE, mZoomIncludeClosestVehicle);
+
+        Location centerLocation = mFragment.getMapView().getMapCenterAsLocation();
+        outState.putDouble(MapParams.CENTER_LAT, centerLocation.getLatitude());
+        outState.putDouble(MapParams.CENTER_LON, centerLocation.getLongitude());
+        outState.putFloat(MapParams.ZOOM, mFragment.getMapView().getZoomLevelAsFloat());
+        Log.v("save zoomlevel", mFragment.getMapView().getZoomLevelAsFloat() + "");
+    }
+
+    @Override
+    public void onViewStateRestored(Bundle savedInstanceState) {
+        String stopId = savedInstanceState.getString(MapParams.STOP_ID);
+        if (stopId == null) {
+            // If there is no focused stop then restore the map state otherwise
+            // let the BaseMapFragment to handle map state with focused stop
+
+            float mapZoom = savedInstanceState.getFloat(MapParams.ZOOM, MapParams.DEFAULT_ZOOM);
+            if (mapZoom != MapParams.DEFAULT_ZOOM) {
+                mFragment.getMapView().setZoom(mapZoom);
+            }
+
+            double lat = savedInstanceState.getDouble(MapParams.CENTER_LAT);
+            double lon = savedInstanceState.getDouble(MapParams.CENTER_LON);
+            if (lat != 0d && lon != 0d) {
+                Location location = LocationUtils.makeLocation(lat, lon);
+                mFragment.getMapView().setMapCenter(location, true, false);
```

Comment:
Yes, we should change this to not animate as well - could you change this to `false` and push as a new commit in this PR, and reference this comment from it?  This way we keep behavior consistent.